### DISCUSSION TITLE
What `flags` stands for in `uv_random`?

### DISCUSSION THREAD

**Discussion by juanarbol** [libuv/libuv#3508] [General]
I saw this code:

```c
# include/uv.h
UV_EXTERN int uv_random(uv_loop_t* loop,
                        uv_random_t* req,
                        void *buf,
                        size_t buflen,
                        unsigned flags,  /* For future extension; must be 0. */
                        uv_random_cb cb);
```

And I would like to implement the `flags` arg, but I don't understand what they should do. A helping hand would be fantastic for supporting this in libuv.

**Comment by bnoordhuis** [comment]
It's not used right now, it's an escape hatch for when a new use case presents itself. Hypothetical example: Linux 6.0 gains a F_QUANTUM_RANDOM flag.
